src/Commands/ValidatorsCommands.php

```diff
@@ -1,135 +0,0 @@
-<?php
```

Comment:
This file was removed in error? These hooks are still in use.

Reply:
No, I removed them from FieldCreateCommands.

Reply:
Validation of entityType and bundle was already happening in the command code, so I removed the validation hooks.

Reply:
@validate-entity-load hook is used by config commands, for example. It cant be removed in this PR.

Reply:
My bad, I didn't realise the ValidatorsCommands already existed before. I'll fix this.

Reply:
Fixed!